CRM/Activity/BAO/Query.php

```diff
@@ -335,11 +334,11 @@ public static function whereClauseSingle(&$values, &$query) {
 
       case 'parent_id':
         if ($value == 1) {
-          $query->_where[$grouping][] = "parent_id.parent_id IS NOT NULL";
+          $query->_where[$grouping][] = "civicrm_activity.parent_id IS NOT NULL";
           $query->_qill[$grouping][] = ts('Activities which have Followup Activities');
         }
         elseif ($value == 2) {
-          $query->_where[$grouping][] = "parent_id.parent_id IS NULL";
+          $query->_where[$grouping][] = "civicrm_activity.parent_id IS NULL";
```

Comment:
This is a fix of a typo present in Activity search query. Leads to an error when a search is made on `is a Followup Activity?` field.